dq-game/src/utils/socket_connection.py

```diff
@@ -1,56 +1,58 @@
-import sys
-import os
-from events.event_handler import EventHandler
 import importlib.util
-import socket
-import time
+import os
 import pickle
-import threading
 import select
+import socket
+import sys
+import threading
+import time
+
+from events.event_handler import EventHandler
+
 from utils.message import Message
 
 # from game_types.question import DQQuestion
 
 
 class SocketConnection(EventHandler):
-    def __init__(self, port):
+    # TODO: compose event handler
```

Comment:
I changed the encoding to use python `pickle` serialization module. This will allow us to send any object that we want through the socket.